bin/pbr/materials/marble/About these PBR files.txt

```diff
@@ -0,0 +1 @@
+These texture files were created by FreePBR.com and may be used freely in your video games and 3d work at no cost. They may not however be redistributed on other websites or anywhere else other than FreePBR.com. We think that is more than fair. :) We also would greatly appreciate it if some sorrt of credit was given if you do indeed use these textures in a published game. Other than that, keep on creating and have fun. :)
```

Comment:
I guess we should remove these text files and mention freePBR in the readme instead